skaffold.yaml

```diff
@@ -49,19 +49,8 @@ build:
   local:
     useBuildkit: false
 deploy:
-  kubectl:
-    manifests:
-    - ./kubernetes-manifests/adservice.yaml
-    - ./kubernetes-manifests/cartservice.yaml
-    - ./kubernetes-manifests/checkoutservice.yaml
-    - ./kubernetes-manifests/currencyservice.yaml
-    - ./kubernetes-manifests/emailservice.yaml
-    - ./kubernetes-manifests/frontend.yaml
-    - ./kubernetes-manifests/paymentservice.yaml
-    - ./kubernetes-manifests/productcatalogservice.yaml
-    - ./kubernetes-manifests/recommendationservice.yaml
-    - ./kubernetes-manifests/redis.yaml
-    - ./kubernetes-manifests/shippingservice.yaml
+  kustomize:
+    paths: [kustomize]
```

Comment:
So we are changing where Skaffold will grab the manifests, now with Kustomize?

Reply:
Sorry for the back and forth here.  I've separated this out into a #1232 with a new kustomization.yaml for local development so that `kustomize/base` is reserved for release manifests, but we can still use kustomize for local development.

Reply:
Thank you for splitting the `skaffold.yaml` stuff into a separate pull-request. Resolving this comment thread now. Let's resume `skaffold`-related discussions in https://github.com/GoogleCloudPlatform/microservices-demo/pull/1232.